app/containers/AccountProvider/saga.js

```diff
@@ -304,6 +312,31 @@ export function* updateAccWorker({ eos }) {
   }
 }
 
+export function* getCommunityPropertyWorker(profile) {
+  try {
+    if (single) {
+      const profileInfo = profile || (yield select(makeSelectProfileInfo()));
+      const eosService = yield select(selectEos);
+
+      const info = yield call(
+        eosService.getTableRow,
+        ALL_PROPERTY_COMMUNITY_TABLE,
+        ALL_PROPERTY_COMMUNITY_SCOPE,
+        profileInfo.user,
+      );
+
+      if (info) {
+        const isAdmin = isUserAdmin(info.properties);
+
+        if (isAdmin) {
+          yield put(getUserProfileSuccess({ ...profileInfo, isAdmin: true }));
+        }
+      }
+    }
+    // eslint-disable-next-line no-empty
+  } catch (e) {}
```

Comment:
Is it expected to hide error here? Is it not critical?

Reply:
Yeah, it's not critical